google-cloud-clients/google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/BigtableColumn.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.bigquery;
+
+
+import com.google.common.base.Function;
+import com.google.auto.value.AutoValue;
+import javax.annotation.Nullable;
+
+@AutoValue
+public abstract class BigtableColumn {
+
+  private static final long serialVersionUID = 1L;
+
+  @Nullable
+  public abstract String getQualifierEncoded();
+
+  @Nullable
+  public abstract String getFieldName();
+
+  @Nullable
+  public abstract Boolean getOnlyReadLatest();
+
+  @Nullable
+  public abstract String getEncoding();
+
+  @Nullable
+  public abstract String getType();
+
+  @AutoValue.Builder
+  public abstract static class Builder {
+
+
```

Comment:
The whitespaces do not conform to Java style (this repo doesn't conform in general, but we're trying to make it). Could you run google-java-format on this?

Reply:
Yanking these manually, but my IDE formatter doesn't seem to take action on these (or its not propagating all the formatting rules through from the plugin).